src/test/java/com/solvd/shop24/BaseTest.java

```diff
@@ -0,0 +1,17 @@
+package com.solvd.shop24;
+
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.solvd.shop24.gui.pages.shop24.HomePage;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+public class BaseTest extends AbstractTest {
+
+    protected HomePage openHomePage(WebDriver driver) {
+        HomePage homePage = new HomePage(driver);
+        homePage.open();
+        driver.manage().window().fullscreen();
+        Assert.assertTrue(homePage.isPageOpened(), "homePage of shop24 was not opened!");
```

Comment:
in fact this action `Assert.assertTrue(homePage.isPageOpened()` is useless here. it just checks if url is correct. but you just opened this page by direct url